sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2392,5 +2392,14 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
         }
       }
     }
+
+    test("load command for non local invalid path validation") {
```

Comment:
Please move this test case out of 
```
Seq(true, false).foreach { caseSensitive =>
  ...
}
```

Reply:
Right gatorsmile, this is why test case was failing, one more change i have done,Since i am dealing with hive table i moved this test case to HiveDDLSuite.scala, hope its fine. Thanks for the review